test/unit/rules.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+
+
+
+  <script src="../../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+  <link rel="import" href="../../polymer.html">
+  <script src="/bower_components/test-fixture/test-fixture-mocha.js"></script>
+  <link rel="import" href="/bower_components/test-fixture/test-fixture.html">
+  <link rel="import" href="/bower_components/iron-test-helpers/iron-test-helpers.html">
```

Comment:
Is using text-fixture not generally considered good practice? I didn't use it because I needed it; I used it because I thought it was recommended. I did notice that all the other test weren't using it but I just figured the test where written pre-dating test-fixture.